style.css

```
@@ -0,0 +1,13 @@
+@charset "utf-8";
+#sikaku1 {
+    width: 256px;
+    height: 256px;
+    background-color: #FF0000;
+}
+
+#sikaku2 {
+    width: 256px;
+    height: 256px;
+    background-color: #004EFF;
+    margin-top: 11px;
+}
```

余談ですが、
```suggestion
#sikaku1, #sikaku2 {
    width: 256px;
    height: 256px;
}

#sikaku1 {
   background-color: #FF0000;
}

#sikaku2 {
    background-color: #004EFF;
    margin-top: 11px;
}
```
とまとめて書くことができます！